.github/workflows/ci.yml

```
@@ -580,12 +586,9 @@ jobs:
           echo "Authorizing minters"
           python3 "$SCRIPT_DIR/../local-network/authorize-minters.py"
 
-          "$BIN_DIR/mc-consensus-tool" wait-for-quiet \
-              insecure-mc://localhost:3200/ \
-              insecure-mc://localhost:3201/ \
-              insecure-mc://localhost:3202/ \
-              insecure-mc://localhost:3203/ \
-              insecure-mc://localhost:3204/
+          echo "Waiting for quiet after authorizing minters..."
+          PRE_MINT_BLOCK_INDEX=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet)
+          echo "Done waiting, PRE_MINT_BLOCK_INDEX=${PRE_MINT_BLOCK_INDEX}"
```

:wrench: building off of @garbageslam's comment I would push harder to disambiguate index vs count
```suggestion
          PRE_MINT_BLOCK_COUNT=$("$BIN_DIR/mc-consensus-tool" wait-for-quiet)
          echo "Done waiting, PRE_MINT_BLOCK_COUNT=${PRE_MINT_BLOCK_COUNT}"
```

---

His comment says `I guess I have a nit here, I think the tool returns block indices, and --beyond-block takes a block index, but the variables are called counts` - the variables should be INDEX which is what I changed them to...

---

oh ok my bad, thanks for the clarification